### PR TITLE
Changelog in preparation for 10.0.8RC2

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

**Comment by PVince81** [comment]
removed reverted fix: https://github.com/owncloud/core/issues/31150#issuecomment-381937569

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,16 +31,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 - Added Symfony events for group membership events - [#31003](https://github.com/owncloud/core/issues/31003)
+- Added Symfony events for feature change in group admin - [#31132](https://github.com/owncloud/core/issues/31132)
 - Added config.php option to select apps to ignore missing signature file (mostly for themes) - [#30891](https://github.com/owncloud/core/issues/30891) [#31066](https://github.com/owncloud/core/issues/31066)
 - Added ability for full-page frontend-only apps in info.xml - [#30918](https://github.com/owncloud/core/issues/30918)
 - Files app UI now retries chunk uploads in web UI on stalled uploads - [#31005](https://github.com/owncloud/core/issues/31005)
-- More user-friendly email address input in link share dialog - [#30945](https://github.com/owncloud/core/issues/30945)
+- More user-friendly email address input and handling in link share dialog - [#30945](https://github.com/owncloud/core/issues/30945) [#31142](https://github.com/owncloud/core/issues/31142)
 
 ### Changed
+- Set minimum php version to 5.6 im composer.json - [#31100](https://github.com/owncloud/core/issues/31100)
```
Comment (phil-davis) [inline CHANGELOG.md]: s/im/in

**Review by phil-davis — APPROVED** [review]
Just 1 little typo - you can fix that I'm sure.

**Comment by PVince81** [comment]
fixed typo, merging

**Pull request merged**